api_app/event_grid/event_sender.py

```diff
@@ -28,14 +28,14 @@ async def send_status_changed_event(airlock_request: AirlockRequest, previous_st
 async def send_airlock_notification_event(airlock_request: AirlockRequest, emails: Dict):
     request_id = airlock_request.id
     status = airlock_request.status.value
-    short_workspace_id = airlock_request.workspaceId[-4:]
+    workspace_id = airlock_request.workspaceId
     snake_case_emails = {re.sub(r'(?<!^)(?=[A-Z])', '_', role_name).lower(): role_id for role_name, role_id in emails.items()}
 
     airlock_notification = EventGridEvent(
         event_type="airlockNotification",
-        data=AirlockNotificationData(request_id=request_id, event_type="status_changed", event_value=status, emails=snake_case_emails, workspace_id=short_workspace_id).__dict__,
+        data=AirlockNotificationData(request_id=request_id, event_type="status_changed", event_value=status, emails=snake_case_emails, workspace_id=workspace_id).__dict__,
```

Comment:
Just for my info - what have we used this value before this change?

Reply:
There was a short workspace URL, which only included the  last 4 characters of the workspace.  (Same is true for StatusChanged event).

I'm not sure why, though! Do you know? To me it makes more sense to send a full workspace ID in the event as the consumer can always take the last 4 characters of it.